source/channels/favorite-channels.rst

```diff
@@ -4,9 +4,10 @@ Mark channels as favorites
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-Marking channels as a favorite is a great way to organize your channel sidebar by choosing which public channels, private channels, and direct messages are most important to you.
+By deciding which public channels, private channels, and direct messages are most significant to you, you can organise your channel sidebar by marking the channels that are most important to you as **Favourites**.
```

Comment:
```suggestion
If you have public channels, private channels, and direct messages that are important to you, mark them as **Favorite** to organize them into the Favorite category your channel sidebar.
```

Reply:
I see where you're going with the change and appreciate it! 

I've added a proposal to the change which includes spelling corrections as we use American English and not UK English, hence the "z" instead of an "s" and no "u" in the word "favorite".